crates/net/network/src/peers/manager.rs

```diff
@@ -740,6 +758,24 @@ impl PeerConnectionState {
     }
 }
 
+/// Represents the kind of peer
+#[derive(Debug, Clone, Copy, Default, Eq, PartialEq)]
+enum PeerKind {
+    /// Non-trusted peer.
+    #[default]
+    NonTrusted,
```

Comment:
we may introduce more types, so perhaps we rename this to something like `Basic`